client/app/billing/autoRenew/ssh/user-ssh.service.js

```diff
@@ -22,10 +22,18 @@ angular.module('UserAccount').service('UseraccountSshService', [
           promises.push(this.getCloudSshList());
       }
 
-      return $q.all(promises).then((data) => {
-        const keys = _.flatten(_.values(data));
-        return _.sortBy(keys, key => key.keyName.toLowerCase());
-      });
+      return $q.allSettled(promises)
+        .then(sshKeys => [sshKeys])
+        .catch(([sshKeys, error]) => (_.isArray(sshKeys) ? [sshKeys, error] : [[], error]))
+        .then(([sshKeys, error]) => {
```

Comment:
Won't it set the second key in `error` ?

Reply:
You might be right, I think I misunderstood the way `allSettled` works. checking right away

Reply:
I misunderstood `allSettled` in that context. Now fixed